client/src/components/Modal/Modal.js

```diff
@@ -1,14 +1,20 @@
 //Modal needs to know about QuestionButton and/or get data from question button
-import React, { useState } from "react";
+import React, {useState, useEffect} from "react";
 import Modal from "react-bootstrap/Modal";
 import Button from "react-bootstrap/Button";
 import Timer from "../Timer";
-import questions from "../../Json/questions.json";
 
 function JeopardyModal(props) {
   // Flag to change contents of modal after timer expires
   const [isExpired, setIsExpired] = useState(false);
 
+  // Reset modal state whenever it is reopened
+  useEffect(() => {
```

Comment:
This is unrelated to the question you asked me, but I noticed that once the modal expired once, it was expired forever. This resets the timer whenever the modal is opened.